### PR TITLE
ruff: apply rule RUF100

### PR DESCRIPTION
**Pull request by Mic92**
part of https://github.com/buildbot/buildbot/pull/8049

### CODE REVIEW

File: www/nestedexample/buildbot_nestedexample/__init__.py [hunk 1]
```diff
@@ -1,3 +1,6 @@
+import functools
```
Comment (p12tic) [inline www/nestedexample/buildbot_nestedexample/__init__.py:1]: Seems like unnecessary change

Comment (Mic92) [reply www/nestedexample/buildbot_nestedexample/__init__.py:1]: Could be an artifact of taking commits apart.

Comment (Mic92) [reply www/nestedexample/buildbot_nestedexample/__init__.py]: fixed.
